### PR TITLE
Add WebDB

### PR DESCRIPTION
**Pull request by aleluff**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Comment by aleluff** [comment]
I just redo entirely the PR

**Review by kokomo123 — APPROVED** [review]
This looks great! Thank you for redoing the PR and making the changes necessary. LGTM.

File: software/webdb.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+tags:
+  - Database Management
+  - Software Development - IDE & Tools
+  - Self-hosting Solutions
+depends_3rdparty: false
+demo_url: https://demo.webdb.app/
```
Comment (Ki-er) [inline software/webdb.yml:14]: This is optional so can be removed.
```suggestion
demo_url: https://demo.webdb.app/
```

File: software/webdb.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+source_code_url: https://gitlab.com/web-db/app
+description: Open Source and Efficient Database IDE.
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+tags:
+  - Database Management
+  - Software Development - IDE & Tools
+  - Self-hosting Solutions
```
Comment (Ki-er) [inline software/webdb.yml:12]: I dont think this fits this tag as its more for all in one solutions.

```suggestion
  - Database Management
  - Software Development - IDE & Tools
```

File: software/webdb.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+name: WebDB
+website_url: https://webdb.app
+source_code_url: https://gitlab.com/web-db/app
+description: Open Source and Efficient Database IDE.
```
Comment (Ki-er) [inline software/webdb.yml:4]: Cleaner description that fits sentence case.
```suggestion
description: Efficient database IDE.
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Couple changes that could be made.

**Comment by aleluff** [comment]
Ok for you now ?

**Review by Ki-er — APPROVED** [review]
LGTM - Thanks!

**Review by nodiscc — APPROVED** [review]

**Comment by aleluff** [comment]
@Ki-er Sorry, I missclick to re-approval buttons

Can I fo something more to make it merge ?

**Comment by Ki-er** [comment]
>  You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

This will be merged soon. Please be patient.

**Comment by aleluff** [comment]
Ok my Bad. Thanks

**Comment by aleluff** [comment]
👋🏻

**Comment by aleluff** [comment]
🙏

**Pull request merged**